mteb/tasks/PairClassification/multilingual/RTE3.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks import MultilingualTask
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGS = {
+    "de": ["deu-Latn"],
+    "en": ["eng-Latn"],
+    "fr": ["fra-Latn"],
+    "it": ["ita-Latn"],
+}
+
+
+class RTE3(MultilingualTask, AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="RTE3",
+        dataset={
+            "path": "maximoss/rte3-multi",
+            "revision": "d94f96ca5a6798e20f5a77e566f7a288dc6138d7",
+        },
+        description="Recognising Textual Entailment Challenge (RTE-3) aim to provide the NLP community with a benchmark to test progress in recognizing textual entailment",
+        reference="https://aclanthology.org/W07-1401/",
+        category="s2s",
+        type="PairClassification",
+        eval_splits=["test", "validation"],
+        eval_langs=_LANGS,
+        main_score="ap",
+        date=("2023-03-25", "2024-04-15"),
+        form=["written"],
+        domains=None,
+        task_subtypes=["Claim verification"],
+        license="cc-by-4.0",
+        socioeconomic_status=None,
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="created",
+        bibtex_citation="""@inproceedings{giampiccolo-etal-2007-third,
+            title = "The Third {PASCAL} Recognizing Textual Entailment Challenge",
+            author = "Giampiccolo, Danilo  and
+            Magnini, Bernardo  and
+            Dagan, Ido  and
+            Dolan, Bill",
+            booktitle = "Proceedings of the {ACL}-{PASCAL} Workshop on Textual Entailment and Paraphrasing",
+            month = jun,
+            year = "2007",
+            address = "Prague",
+            publisher = "Association for Computational Linguistics",
+            url = "https://aclanthology.org/W07-1401",
+            pages = "1--9",
+        }
+        """,
+        n_samples={"validation": 3200, "test": 3200},
+        avg_character_length={"validation": 146.0, "test": 126.1},
+    )
+
+    def load_data(self, **kwargs):
+        """Load dataset from HuggingFace hub"""
+        if self.data_loaded:
+            return
+        self.dataset = datasets.load_dataset(
+            self.metadata.dataset["path"], revision=self.metadata.dataset["revision"]
+        )
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def dataset_transform(self):
+        _dataset = {}
+        for lang in self.langs:
+            _dataset[lang] = {}
+            for split in self.metadata.eval_splits:
+                # keep target language
+                hf_dataset = self.dataset[split].filter(lambda x: x["language"] == lang)
+                # keep labels 0=entailment and 2=contradiction, and map them as 1 and 0 for binary classification
```

Comment:
Is it not possible to do three labels for pair clf?

Reply:
I look into the other datasets (e.g. XNLI) and got the impression that binary classification was a requirement. Probably for metrics like average precision.

Reply:
Yep for pair classification, the evaluator only handles binary labels, see here:
https://github.com/embeddings-benchmark/mteb/blob/03e05c8cf702352af9fde34107cb63b79d12c93d/mteb/evaluation/evaluators/PairClassificationEvaluator.py#L27